appveyor.yml

```diff
@@ -35,6 +35,7 @@ install:
 
   # Check pip version
   - pip --version
+  - python -m pip install --upgrade pip setuptools wheel
```

Comment:
Might be nice to move this command one line up such that `pip --version` shows the new version, after pip was upgraded.

Reply:
Okay.